src/OnlineRetailPortal.Web/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace OnlineRetailPortal.Web.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
Do we need this

Reply:
removed WeatherForecastController
git does not push empty folder so I had created class1 is emptyclasss so git will push those empty folders to repo